probes/scripts/livenessProbe.sh

```diff
@@ -6,7 +6,7 @@
 #
 RETVAL=$(test -f /operator/debug-config/livenessProbeSuccessOverride ; echo $?)
 
-FILE=/operator/.alive
+FILE=/probes/.alive
```

Comment:
I assume this file is now used by both operator and webhook? If so, then the error message for a missing liveness probe does not match the name of this file - at least in the webhook case - not sure about the operator case.

Reply:
I'm not sure which error message are referring to, can you please clarify?